### PR TITLE
Use Mjolnir's client for logMessage since it is available.

### PR DESCRIPTION
**Pull request by Gnuxie**

### CODE REVIEW

File: src/Mjolnir.ts [hunk 1]
```diff
@@ -343,13 +343,12 @@ export class Mjolnir {
         if (!additionalRoomIds) additionalRoomIds = [];
         if (!Array.isArray(additionalRoomIds)) additionalRoomIds = [additionalRoomIds];
 
-        if (config.RUNTIME.client && (config.verboseLogging || LogLevel.INFO.includes(level))) {
```
Comment (ShadowJonathan) [inline src/Mjolnir.ts:346]: Wasn't this done likt this to make sure that mjolnir still correctly logs even when the Matrix Client fails to initialize?

Comment (Gnuxie) [reply src/Mjolnir.ts:346]: That's here https://github.com/matrix-org/mjolnir/blob/main/src/index.ts#L45-L69

Comment (ShadowJonathan) [reply src/Mjolnir.ts:346]: Ah alright, then this can be closed

**Review by jesopo — APPROVED** [review]

**Comment by Gnuxie** [comment]
Ok looks like this is covered by #334

**Pull request closed without merging**